requirements-test.txt

```diff
@@ -32,7 +32,7 @@ colorama==0.4.1
 constantly==15.1.0
 coverage==4.5.4
 cssselect==1.1.0
-dask==2.3.0
+dask
```

Comment:
@alexgleith For evironment setup not a fan of unpinned versions...

Reply:
Yeah, I should have caught that. I was keeping all of these pinned for Travis stability, with pyup configured [to bump them periodically](https://github.com/jeremyh/datacube-explorer/pull/35). But I stopped looking at pyup's PRs, so they've falled behind lately.